reframe/core/launchers/__init__.py

```diff
@@ -73,8 +82,6 @@ def command(self, job):
     def run_command(self, job):
         '''The full launcher command to be emitted for a specific job.
 
-        This includes any user options.
```

Comment:
I think this is still accurate:

https://github.com/reframe-hpc/reframe/blob/cb4a2ac751c0f19637d735e4009dd20518d3a2af/reframe/core/launchers/__init__.py#L86

Maybe we should be more specific by writing "user-specific launcher options."